src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -23,12 +25,17 @@ servers:
     description: 'Version 1'
 
 paths:
-  # Internet Imageing Protocol
+  # Internet Imaging Protocol
   /IIP/:
     get:
       tags:
         - 'Access'
-      summary: 'Internet Imaging Protocol '
+      summary: 'Internet Imaging Protocol'
+      description: |-
+        Most of our services at the Royal Library makes use of the IIIF standard.
+        However some of our services require the IIP protocol. 
+        This service (IIP) is in use but support may be dropped in the future.
+        There is no support or updates scheduled for this endpoint.
```

Comment:
Reads a bit too much as an excuse. Could you simplify it so that is is not so much about us, but more about the user? Something like "IIP is currently supported for backwards compatibility and support might be dropped in the future... (including the last bit about no scheduled updates)."